build/assets/css/main.css

```diff
@@ -2,105 +2,219 @@
 html {
   color: #222;
   font-size: 1em;
-  line-height: 1.4; }
-
+  line-height: 1.4
```

Comment:
Things like color choices to maintain consistent palates right? I agree. We did, to a large extent. This is the `main.css`, not the `main.scss` file. We're not using the `main.css` file, as it is the version compiled from the `main.scss`.
Moving forward though, it'd be super-helpful to bear your advice in mind. 👍